### PR TITLE
Add helm release reinstall procedure to docs

### PR DESCRIPTION
**Pull request by stefanprodan**
Document how to force a helm release install when an upgrade fails.

### CODE REVIEW

File: site/helm-integration.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ menu_order: 90
   * [The `HelmRelease` custom resource](#the-helmrelease-custom-resource)
     + [Using a chart from a Git repo instead of a Helm repo](#using-a-chart-from-a-git-repo-instead-of-a-helm-repo)
       - [Notifying Helm Operator about Git changes](#notifying-helm-operator-about-git-changes)
+      - [Reinstalling a Helm release](#reinstalling-a-helm-release)
```
Comment (hiddeco) [inline site/helm-integration.md]: I think this should be one level higher (as it is not related to charts from Git repositories).

Comment (stefanprodan) [reply site/helm-integration.md]: Hmm the `using-a-chart-from-a-git-repo-instead-of-a-helm-repo` doesn't even exists

File: site/helm-integration.md [hunk 2]
```diff
@@ -119,6 +120,20 @@ OK
 > either need to port forward before making the request or put something
 > in front of it to serve as a gatekeeper.
 
+#### Reinstalling a Helm release
```
Comment (hiddeco) [inline site/helm-integration.md]:
```suggestion
### Reinstalling a Helm release
```

**Comment by hiddeco** [comment]
Needs a rebase due to our new merge rules.

**Pull request merged**